src/src/PluginStructs/P014_data_struct.cpp

```diff
@@ -175,148 +249,279 @@ uint8_t P014_data_struct::checkCRC(uint16_t data, uint8_t check)
   return (uint8_t)remainder;
 }
 
+
+
+
+
 /* ======================================================================
-   Function: si7021_readRegister
-   Purpose : read the user register from the sensor
-   Input   : user register value filled by function
-   Output  : 0 if okay
-   Comments: -
-   ====================================================================== */
-int8_t P014_data_struct::readRegister(uint8_t *value)
+Function: Plugin_123_si7013_setResolution
```

Comment:
This comment is not correct as it is not Plugin 123

Reply:
sorry. A left over. ;)